src/madsci_common/madsci/common/types/node_types.py

```diff
@@ -368,6 +368,11 @@ class NodeStatus(MadsciBaseModel):
         title="Node Errored",
         description="Whether the node is in an errored state.",
     )
+    disconnected: bool = Field(
```

Comment:
Let's make sure we update the ready and description computed fields to include this field as well.